### PR TITLE
Upgraded the TUtoolbox to 2023.0.0.666

### PR DESCRIPTION
**Pull request by chbwien**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94711)

### CODE REVIEW

**Comment by chbwien** [comment]
That's odd. The message says:
Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?
And the checker says, only 1.2.0 is supported....

File: manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.locale.de-DE.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Author: Christian Beck
+PackageName: TUtoolbox
+PackageUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox
+License: Proprietäre Software von TU.it, TU Wien
+# LicenseUrl: 
+Copyright: © 2021-2023 TU.it, TU Wien
+# CopyrightUrl: 
+ShortDescription: Ein Programm zur Installation aller benötigten Werkzeuge und Patches für TU-Mitglieder.
+ReleaseNotesUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/index.html#history
+Moniker: tutoolbox
```
Comment (mdanish-kh) [inline manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.locale.de-DE.yaml:19]:
```suggestion
```
Only the `ManifestType: defaultLocale` supports Moniker field

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
> That's odd. The message says: Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)? And the checker says, only 1.2.0 is supported....

That's a mistake, please ignore that message. The PR template was modified a bit too early.

File: manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TUitTUWien.TUtoolbox
+PackageVersion: 2023.0.0.666
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Installers:
+  - Architecture: neutral
```
Comment (mdanish-kh) [inline manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.installer.yaml:9]:
```suggestion
  - Architecture: x86
```
Neutral includes all architectures including arm and arm64

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/94711#discussion_r1088160460

File: manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.installer.yaml (hunk 2 above)
Comment (chbwien) [reply manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.installer.yaml:9]: That's right. And it works tested on them. So need to change.

File: manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.locale.de-DE.yaml (hunk 1 above)
Comment (chbwien) [reply manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.locale.de-DE.yaml]: changed it with the current push

File: manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.installer.yaml (hunk 2 above)
Comment (mdanish-kh) [reply manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.installer.yaml:9]: @chbwien We've actually move away from using `Neutral` as an architecture type. Best practice is that if it works on all platforms, then you should create separate Installer nodes for them as such

```yaml
PackageIdentifier: TUitTUWien.TUtoolbox
PackageVersion: 2023.0.0.666
InstallerType: inno
InstallerSwitches:
  Custom: /suppressmsgboxes
MinimumOSVersion: 10.0.0.0
Scope: machine
Platform:
  - Windows.Desktop
UpgradeBehavior: install
Installers:
- Architecture: x64
  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2023_0_0_666.exe
  InstallerSha256: 64bc1e36cc2d980bab084980cf96581ab6b825f9dbee41954f1179e615b01d18
- Architecture: x86
  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2023_0_0_666.exe
  InstallerSha256: 64bc1e36cc2d980bab084980cf96581ab6b825f9dbee41954f1179e615b01d18
- Architecture: arm
  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2023_0_0_666.exe
  InstallerSha256: 64bc1e36cc2d980bab084980cf96581ab6b825f9dbee41954f1179e615b01d18
- Architecture: arm64
  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2023_0_0_666.exe
  InstallerSha256: 64bc1e36cc2d980bab084980cf96581ab6b825f9dbee41954f1179e615b01d18
ManifestType: installer
ManifestVersion: 1.2.0
```
You can use this as your installer manifest. I've moved the common fields outside as they get inherited in each installer node. Also you don't need to pass Silent and SilentWithProgress switches as they get automatically passed to installer when installer type is set to inno.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**